### DISCUSSION TITLE
ROMS on TF1/INT

### DISCUSSION THREAD

**Discussion by jonasjacob91** [eduardofilo/RG350_adam_image#119] [SD topics]
Is it possible to partition the unused space on TF1 and mount it as the Partition for the roms?
I managed to rename the Partition as "sdcard", so it is recognized and mounted in /media/sdcard. That works for the handheld.
But lower case fat32-partition names works only for linux, not windows.

I`ve tried to edit fstab or create a symlink to the right partition, but filesystem is read only

**Comment by eduardofilo** [comment]
Yes. Many combinations are possible. After all, OpenDingux is a Linux distribution. But I don't think it's worth creating another partition on the internal card. I would expand partition 2 to occupy all the space on the card and simply install the ROMs there, in the path `/media/data/roms` for example. Then SimpleMenu would have to be reconfigured accordingly.

**Comment by jonasjacob91** [comment]
Thanks for your quick reply, i will try that later.

I was searching for an alternative way instead of reconfiguring SimpleMenu, Retroarch etc. and i would prefer inserting the sd card to windows to transfer roms.

I thought it is the easiest way to change the mounting point of the new created partition to /media/sdcard. 
I didn’t find a password for ssh into the device as root. Is the root password secret?

**Reply by eduardofilo** [reply]
If all of these consoles didn't have two slots, I would have tried something similar too. At least configure the paths for the ROMs to the internal card. Although I use Linux and I don't have much need for the partition for the ROMs to be FAT.

You can change the password or disable it with the Password application. The default password is indicated in this section of Readme:
https://github.com/eduardofilo/RG350_adam_image/#system-access
